atlasdb-api/src/main/java/com/palantir/atlasdb/versions/AtlasDbVersion.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.versions;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.CharStreams;
+
+public final class AtlasDbVersion {
+    public static final String REPORT_VERSION_PROPERTY = "atlasdb.report.version";
+    public static final String REPORT_VERSION_DEFAULT = "true";
+    public static final String VERSION_UNKNOWN_STRING = "VERSION UNKNOWN";
+    public static final String VERSION_INFO_FILE = "/META-INF/atlasdb.version";
+
+    private static volatile boolean versionPrinted = false;
+
+    private AtlasDbVersion() {
+        // static class
+    }
+
+    public static void reportVersion() {
+        reportVersion(System.out);
+    }
+
+    public static void reportVersion(PrintStream writer) {
+        if (!versionPrinted && Boolean.parseBoolean(System.getProperty(REPORT_VERSION_PROPERTY, REPORT_VERSION_DEFAULT))) {
+            String version = readVersion();
+            System.out.println("AtlasDB Version: " + version);
```

Comment:
I kind of want to keep it around, just in case. We might want to log this to a file at some point in the future